src/pkg/oidc/testutils.go

```diff
@@ -2,11 +2,9 @@ package oidc
 
 import (
 	"context"
+	"errors"
 	"fmt"
 	"strconv"
-)
```

Comment:
This close parentheses should NOT be removed.

Reply:
Seems linter's bug, let me fix it.